FusionIIIT/applications/academic_information/models.py

```diff
@@ -274,6 +274,8 @@ class Calendar(models.Model):
     '''
     from_date = models.DateField()
     to_date = models.DateField()
+    start = models.TimeField(default='20:00')
+    end = models.TimeField(default='20:00')
```

Comment:
rename each to start_time & end_time
refer [https://stackoverflow.com/questions/16027516/can-i-set-a-specific-default-time-for-a-django-datetime-field](url)
and modify default accordingly. Also push a single migration file after changes.